iib/web/static/api_v1.yaml

```diff
@@ -181,6 +186,7 @@ paths:
           application/json:
             schema:
               # TODO: update to include Regenerate Bundle attrs
```

Comment:
Should we add regenerate-bundle and merge-index-image?

Reply:
@yashvardhannanavati could you please validate what belongs here?

Reply:
yup, we will have to add the attributes that we call the `patch` endpoint with from the regenerate-bundle and merge-index-image endpoints. I believe they can be found in `build.py` and `build_merge_index_image.py`